### ISSUE TITLE
Reporting - Export of report should contain columns currently in interactive report

### ISSUE THREAD

**Issue by tpurschke** [CactuseSecurity/firewall-orchestrator#2237]
Can we export the report as is?
E.g. if a user hides a column or resorts column order, can we export the report in exactly the form it is currently displayed?

**Comment by tpurschke** [comment]
@NilsPur can you answer the question (not implement the issue)?
A rough estimate of effort (if feasible) would be nice.

**Comment by NilsPur** [comment]
Yes, we can, but we obviously need to integrate the blazor table state into our export function to do so.
By having access to it we can get the colum visibility, name, order and even more.
Changing the behaviour for every report might take a bit though, but probably not more than 1-3 hours.
